docs/source/developer/resources.rst

```diff
@@ -79,31 +81,58 @@ The Python module must contain the following function definitions:
     - ``data``: If ``type`` is ``html``, then a HTML string is expected.
       If type is ``image``, a Pillow Image object is expected.
 
-.. function:: subtitle(request, resource)
+If required, the generator class can contain the class variable ``additional_valid_options`` containing the dictionary of parameter options for the resource.
+For example, a resource may contain the following:
+
+.. code-block:: python
+
+  additional_valid_options = {
+      "display_numbers": [True, False],
+      "black_back": [True, False],
+  }
+
+If ``additional_valid_options`` are given, the ``subtitle`` property method should be overridden.
+The method should display the additional options and also call the parent's subtitle result:
+
+.. function:: subtitle(self)
```

Comment:
Probably worth including that if `subtitle` is overloaded by the subclass, it must be a property (i.e. have the property decorator `@property` and only take one parameter, `self`).